QModManager/Utility/Logger.cs

```diff
@@ -16,13 +17,40 @@ internal static bool EnableDebugging
             }
         }
 
-        internal static void Log(string prefix, string text) => Console.WriteLine($"[QModManager{prefix}] {text}");
-        internal static void Log(string text) => Log("", text);
-        internal static void Debug(string text) { if (EnableDebugging) Log("/DEBUG", text); }
-        internal static void Info(string text) => Log("/INFO", text);
-        internal static void Warn(string text) => Log("/WARN", text);
-        internal static void Error(string text) => Log("/ERROR", text);
-        internal static void Exception(Exception e) => Log("/EXCEPTION", e.ToString());
-        internal static void Fatal(string text) => Log("/FATAL", text);
+        internal static void Log(string prefix, string text)
+        {
+            Console.WriteLine($"[QModManager{prefix}] {text}");
```

Comment:
How should we handle SMLHelper's logging?
I would suggest changing the format to: `[QModManager] [<FROM>] [<LOGLEVEL>]`

Reply:
I am fine with this approach.   
```[QModManager] [<CallingAssemblyName>] [<LOGLEVEL>]```  
I am aware that what were once SMLHelper logs will now be QModManager logs but I am okay with that.

Reply:
By FROM I dont mean Calling Assembly Name. I mean where the log originated from. (Patcher, VersionCheck, SMLHelper, etc)

Reply:
#95